src/ui/pages/work/partusage/PartUsage.js

```diff
@@ -79,14 +68,15 @@ function PartUsage(props) {
                 </Button>
             </div>
             <PartUsageDialog
-                handleSave={handleAddPartUsage}
                 showNotification={props.showNotification}
                 handleError={props.handleError}
                 handleCancel={() => setIsDialogOpen(false)}
                 tabLayout={props.tabLayout}
                 isDialogOpen={isDialogOpen}
                 workorder={props.workorder}
-                isLoading={isLoading}/>
+                isLoading={isLoading}
+                successHandler={successHandler}
+                errorHandler={errorHandler}/>
```

Comment:
There is already a prop passed to `PartUsageDialog` that exposes the `props.handleError` function: `handleError` (line 72 here)